docs/zh-CN/extending-doris/doris-manager/compiling-deploying.md

```diff
@@ -27,15 +27,36 @@ under the License.
 # 编译与部署
 
 ## 编译
-直接运行manager路径下的build.sh脚本，会在manager路径下生成安装运行包——output，其中包括:
-1、Doris Manager的运行包doris-manager.jar
-2、运行的配置文件夹conf
-3、启动脚本start_manager.sh
-4、停止脚本stop_manager.sh
+直接在manager路径下的build.sh脚本
```

Comment:
Update English document together ~

Reply:
The English documentation has not yet been written

Reply:
WIP is ok

Reply:
I just translated the Chinese document through Google, please review it again